From 231a1e1d3ab525272b44bd20f703f9253fd1ed5c Mon Sep 17 00:00:00 2001
From: Ross Burton <ross.burton@arm.com>
Date: Mon, 9 Aug 2021 15:25:52 +0100
Subject: [PATCH 29/30] tests: check for vfat kernel support and tools

t1100-busy-label.sh and t1101-busy-partition.sh create and mount VFAT
partitions, so check for both the tools and the kernel support.

Fixes bug#49594.

Signed-off-by: Brian C. Lane <bcl@redhat.com>
---
 tests/t1100-busy-label.sh     | 3 +++
 tests/t1101-busy-partition.sh | 2 ++
 2 files changed, 5 insertions(+)

diff --git a/tests/t1100-busy-label.sh b/tests/t1100-busy-label.sh
index 71b847c..a77a70f 100755
--- a/tests/t1100-busy-label.sh
+++ b/tests/t1100-busy-label.sh
@@ -19,6 +19,9 @@
 . "${srcdir=.}/init.sh"; path_prepend_ ../parted
 require_root_
 require_scsi_debug_module_
+require_fat_
+require_filesystem_ vfat
+
 ss=$sector_size_
 
 scsi_debug_setup_ sector_size=$ss dev_size_mb=10 > dev-name ||
diff --git a/tests/t1101-busy-partition.sh b/tests/t1101-busy-partition.sh
index 5e37814..c936718 100755
--- a/tests/t1101-busy-partition.sh
+++ b/tests/t1101-busy-partition.sh
@@ -22,6 +22,8 @@ test "$VERBOSE" = yes && parted --version
 
 require_root_
 require_scsi_debug_module_
+require_fat_
+require_filesystem_ vfat
 
 # create memory-backed device
 scsi_debug_setup_ dev_size_mb=10 > dev-name ||
-- 
2.31.1

